client/state/data-layer/wpcom-http/pipeline/retry-on-failure/test/index.js

```diff
@@ -0,0 +1,152 @@
+/**
+ * External dependencies
+ */
+import deepFreeze from 'deep-freeze';
+import { expect } from 'chai';
+import { spy } from 'sinon';
+import { merge } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { useFakeTimers } from 'test/helpers/use-sinon';
+import { http } from 'state/data-layer/wpcom-http/actions';
+import { retryOnFailure as rof } from '../';
+import { noRetry, exponentialBackoff } from '../policies';
+
+const retryOnFailure = rof();
+const retryWithDelay = delay => rof( () => delay );
+
+const nextError = { fail: 'failed big time' };
+
+const getSites = deepFreeze( http( {
+	method: 'GET',
+	path: '/sites',
+	apiVersion: 'v1',
+} ) );
+
+const withRetries = retryCount => actionOrInbound =>
+	undefined !== actionOrInbound.originalRequest
+		? merge( actionOrInbound, { originalRequest: withRetries( retryCount )( actionOrInbound.originalRequest ) } )
+		: merge( actionOrInbound, { meta: { dataLayer: { retryCount } } } );
+
+describe( '#retryOnFailure', () => {
+	let clock;
+	let dispatch;
+	let store;
+
+	useFakeTimers( fakeClock => clock = fakeClock );
+
+	beforeEach( () => {
+		dispatch = spy();
+		store = { dispatch };
+	} );
+
+	it( 'should pass through initially successful requests', () => {
+		const inbound = { nextData: 1, originalRequest: getSites, store };
+
+		expect( retryOnFailure( inbound ) ).to.equal( inbound );
+
+		clock.tick( 20000 );
```

Comment:
where did this number come from? just a number thats greater than when the first retry would have happened?